media/rtmp2segment.go

```diff
@@ -49,17 +49,18 @@ func (ms *MediaSegmenter) RunSegmentation(ctx context.Context, in string, segmen
 			clog.Errorf(ctx, "Stopping segmentation, input stream does not exist. in=%s err=%s", in, err)
 			break
 		}
-		ffmpeg.FfmpegSetLogLevel(ffmpeg.FFLogWarning)
-		_, err = ffmpeg.Transcode3(&ffmpeg.TranscodeOptionsIn{
-			Fname: in,
-		}, []ffmpeg.TranscodeOptions{{
-			Oname:        outFilePattern,
-			AudioEncoder: ffmpeg.ComponentOptions{Name: "copy"},
-			VideoEncoder: ffmpeg.ComponentOptions{Name: "copy"},
-			Muxer:        ffmpeg.ComponentOptions{Name: "segment"},
-		}})
+		cmd := exec.Command("ffmpeg",
```

Comment:
@j0sh what were the reasons to use lpms transcode3 here? Guessing mostly ease of use since it was in go-livepeer already?

Reply:
correct - we don't want to require additional executables outside go-livepeer itself (but this fix is fine for now since it gets us out of a pickle)